resources/webcourses/src/app/pages/webcourse/activities/sidebar/bookmarks/bookmarks.component.html

```diff
@@ -0,0 +1,15 @@
+<div
+  *ngIf="this.bookmarksService.selectedBookmark as bookmarkIndex"
+  @fadeInOut
+  >
+  <div *ngFor="let bookmark of bookmarkIndex">
+    <app-bookmark-card [bookmark]="bookmark"></app-bookmark-card>
+  </div>
+  <p *ngIf="bookmarkIndex.length === 0" class="my-1 text-left ml-2 text-dark">no activities have been bookmarked</p>
```

Comment:
- [ ] The message is missing styling. Copy classes/ids (including padding / margins) from Help sidebar tab to ensure that they also get properly styled when a custom publisher theme is in use.
- [ ] The message does not show up when the Bookmarks tab is clicked, before any activities are bookmarked. Only when bookmarks are added, and then the last is removed that it shows up